themes/default/files/resources/exam.css

```diff
@@ -340,7 +341,9 @@ input.jme {
 .student-answer .feedback-icon {
     margin-left: 0.2em;
 }
-
+.student-answer{
```

Comment:
This applies to every part type. You want something like `.part.type-matrix .student-answer`.